build.ps1

```diff
@@ -353,11 +356,22 @@ try
             Write-Host -ForegroundColor Magenta "IMPORTANT: Using local packages."
         }
 
+        # Tests categories to include
+        $testCategory = "("
+        $testCategory += "TestCategory=E2E"
+        # Invalid Cert tests currently require docker on linux (docker on windows do not allow the same networking setup)
```

Comment:
Is this accurate?

```suggestion
        # Invalid cert tests don't currently work with docker on Windows because of networking requirements and configuration.
```

Reply:
accurate and updated in latest commit